Pax.sln

```diff
@@ -0,0 +1,59 @@
+﻿

```

Comment:
In case you haven't used it before, you might want to experiment with 
git's "cherrypick" feature to tidy up commits.

On 2016-07-14 10:57, TMVector wrote:
> In Pax.sln [1]:
> 
>> @@ -0,0 +1,59 @@
>> +﻿
> 
> This was an accidental add. I will remove it from this PR - I think it
> would be better to add it separately without all my user-specific
> stuff and change/remove build.sh at the same time.
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/9#discussion_r70779301
> [2]
> https://github.com/niksu/pax/pull/9/files/7f29135a9261e8b25f7b9c3ec6cc9f4b3b0add3a..f7ac43cbd665a50aff815bdb5eb1eb81e6b9d99b#r70779301
> [3]
> https://github.com/notifications/unsubscribe/ABDTTg7gHDpESySYbNfbwKf4oaKkQvjuks5qVgf5gaJpZM4JLXUX

-- 
http://www.cl.cam.ac.uk/~ns441/